core/new-gui/src/app/dashboard/service/workflow-version/workflow-version.service.ts

```diff
@@ -23,4 +24,12 @@ export class WorkflowVersionService {
   public workflowVersionsDisplayObservable(): Observable<readonly string[]> {
     return this.workflowVersionsObservable.asObservable();
   }
+
+  public setDisplayParticularVersion(flag: boolean): void {
+    this.displayParticularWorkflowVersion.next(flag);
+  }
```

Comment:
if this method is only used in this service, we shall consider make it private? so all other services/components can only read from this flag.

Reply:
no, it is used by the component to access the private member.